api/specs.proto

```diff
@@ -264,6 +264,9 @@ message ContainerSpec {
 	// task will exit and a new task will be rescheduled elsewhere. A container
 	// is considered unhealthy after `Retries` number of consecutive failures.
 	HealthConfig healthcheck = 16;
+
+	// SecurityProfiles specifies per-architecture security options required by the container
+	SecurityProfiles security_profiles = 21;
```

Comment:
The major problem w/ it being in swarm is the expectation between services.

if a service worked by doing `docker service create --grant=networking my-service`, and we change swarm's mapping server-side, that same cmd-line might not work in a future version of swarm.